<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>historyList</class>
 <widget class="QWidget" name="historyList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>441</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>411</width>
    <height>441</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>411</width>
    <height>441</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>History</string>
  </property>
  <widget class="QListWidget" name="history_listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>221</width>
     <height>401</height>
    </rect>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="currentRow">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="load_pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>405</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Load State</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="timestamp_label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>410</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TIMESTAMP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>0</y>
     <width>181</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Main</string>
   </property>
   <widget class="QLabel" name="iwad_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>25</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>IWAD</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="complevel_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>45</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>COMPLEVEL</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="level_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LEVEL</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="skill_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SKILL</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>90</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Wads</string>
   </property>
   <widget class="QLabel" name="pwads_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>PWAD
PWAD
PWAD</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>190</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Demos</string>
   </property>
   <widget class="QLabel" name="demo_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>RECORD
DEMO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>250</y>
     <width>181</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Extra</string>
   </property>
   <widget class="QLabel" name="extra_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>EXTRA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="reload_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>407</y>
     <width>26</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
image: url(:/pngs/pngs/reload.png);
border-radius:10px;
}
QPushButton::pressed{
	background-color: rgb(127, 129, 129);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
